letsencrypt/config.yaml

```
@@ -116,6 +116,7 @@ schema:
     rfc2136_port: str?
     rfc2136_secret: str?
     rfc2136_server: str?
+    rfc2136_sign_query: bool
```

This should be an optional value, otherwise folks will have to provie the value even if they don't use this feature:

```suggestion
    rfc2136_sign_query: bool?
```